docs/guides/auth.md

```diff
@@ -302,7 +302,26 @@ $client = new helloworld\GreeterClient(
   new Grpc\BaseStub('localhost:50051', []));
 ...
 
-// Authenticating with Google
+// Authenticate using JWT access token (recommended approach)
+function updateAuthMetadataCallback($context)
+{
+    $authUri = $context->service_url;
+    $methodName = $context->method_name;
+    $auth_credentials = ApplicationDefaultCredentials::getCredentials();
+
+    return $auth_credentials->updateMetadata($metadata = [], $authUri);
+}
+$opts = [
+  'credentials' => Grpc\Credentials::createSsl(file_get_contents('ca.pem'));
+];
+
+$client = new helloworld\GreeterClient('localhost:50051', $opts);
```

Comment:
2 issues here:

1. It looks like $opts is kinda optional and that would mean that by default, we create an insecure stub. Am I getting this right?
2. It is unsafe to send google auth tokens to something else that an google endpoint. 'localhost:50051' may be ok since you're sending that locally but:
  - This does not set a good example.
  - You will need some kind of ssl override to make that work.